research_phase/requirements.txt

```diff
@@ -1,9 +1,10 @@
 # We use compatible release functionality (see PEP 440 here: https://www.python.org/dev/peps/pep-0440/#compatible-release)
 # to specify acceptable version ranges of our project dependencies. This gives us the flexibility to keep up with small
 # updates/fixes, whilst ensuring we don't install a major update which could introduce backwards incompatible changes.
-numpy>=1.20.0,<1.21.0
-pandas>=1.3.5,<1.4.0
-scikit-learn>=1.0.2,<1.1.0
-jupyter>=1.0.0,<1.1.0
-feature_engine>=0.3.1,<0.4.0
-matplotlib>=3.1.2,<4.0.0
\ No newline at end of file
+
+numpy>=1.20.0,<1.25.0;python_version>="3.8"
```

Comment:
@jncabelin we may need to say >=3.8 and <3.11. One to keep an eye on as you test